src/verification.rs

```diff
@@ -849,6 +849,11 @@ impl VerificationRequest {
     ///
     /// Will be present only if the other side requested the
     /// verification or if we’re in the ready state.
+    ///
+    /// # Returns
+    ///
+    /// `undefined` if we do not yet know the supported methods; otherwise, an
+    /// array of `VerificationMethod`s.
```

Comment:
this documentation was removed in #82 (https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/82/files#diff-cf8f469cee769b4b3555833df9943f95a94dd55a4c6c5c8b8c7419e34fc8b5d8L879).